template/package.json

```diff
@@ -71,7 +71,9 @@
           "json"
         ],
         "actionItem": {
-          "label": "${!!config.codeIntel.mixPreciseAndSearchBasedReferences && \"Hide search-based results when precise results are available\" || \"Mix precise and search-based results\"}"
+          "label": "${!!config.codeIntel.mixPreciseAndSearchBasedReferences && \"Hide search-based results\" || \"Mix precise and search-based results\"}",
+          "description": "${!!config.codeIntel.mixPreciseAndSearchBasedReferences && \"Hide search-based results when precise results are available\" || \"\"}",
+          "iconURL": "data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20100%20100%22%3E%3Ctext%20y%3D%22.9em%22%20font-size%3D%2290%22%3E%E2%84%B9%EF%B8%8F%3C%2Ftext%3E%3C%2Fsvg%3E"
```

Comment:
I just saw the video in the other PR: I think this should be a material design icon, otherwise it really sticks out.

You could take the SVG from here: https://github.com/sourcegraph/sourcegraph/blob/659e7ceced1b1d7646cc570be27a62a969438cf9/client/wildcard/src/components/Alert/Alert.module.scss#L112-L121

Reply:
Actually Sara and I [agreed to make it without an icon]( https://sourcegraph.slack.com/archives/C03PPPY5T6V/p1660204839248049?thread_ts=1660058606.156389&cid=C03PPPY5T6V). I just cherry-picked the wrong version 🤦‍♂️
Will make another PR now!